scripts/us_census/geojsons_low_res/generate_mcf.py

```diff
@@ -46,33 +47,51 @@ def generate_simple_geojsons(self, verbose=True):
             # TODO(jeffreyoldham): (potentially) find a cleaner way to do this.
             if int(geoid.split('/')[1]) > 100:
                 continue
-            if verbose:
-                logging.info(f"Simplifying {geoid}...")
+            logging.info(f"Simplifying {geoid}...")
             try:
                 with tempfile.TemporaryFile(mode='r+') as f1, \
                      tempfile.TemporaryFile(mode='r+') as f2:
 
                     geojson.dump(coords, f1)
-                    geojson.dump(
-                        self.simplifier.simplify(coords,
-                                                 verbose=verbose,
-                                                 epsilon=0.05), f2)
+                    simple = self.simplifier.simplify(coords, epsilon=0.01)
+                    geojson.dump(simple, f2)
 
                     # Rewind files to start for reading.
                     f1.seek(0)
                     f2.seek(0)
 
                     plotter.compare_plots(gpd.read_file(f1),
                                           gpd.read_file(f2),
-                                          show=False)
+                                          show=show)
+                    self.simple_geojsons[geoid] = simple
             except AssertionError:
                 logging.error("Simplifier failure on GeoJSON below:\n", coords)
         plt.show()
 
-    # def generate_mcf(self, path='low_res_geojsons.mcf'):
-    #     with open(path, 'w') as f:
+    def generate_mcf(self, path='low_res_geojsons.mcf'):
+        """Writes the simplified GeoJSONs to an MCF file.
 
+        Args:
+            path: Path to MCF file to write simplified GeoJSON to.
+        """
+        temp = "\n".join([
+            "Node: dcid:{geoid}",
+            "typeOf: dcs:{type}",
+            "geoJsonCoordinatesLowRes: {coords_str}",
```

Comment:
I wonder if LowRes is the right term here (it's still pretty high res, high enough for display at large sizes). I can't find some standard classification scheme, but found this: https://www.naturalearthdata.com/downloads/

There should be an additional node to define this property - will that be hand written separately for import? For geoJsonCoordinates, its:

```
Node: dcid:geoJsonCoordinates
typeOf: schema:Property
name: "geoJsonCoordinates"
description: "Geographical boundary of a place in GeoJSON format."
domainIncludes: schema:Place
rangeIncludes: schema:Text
```

Reply:
@pradh suggested the name `geoJsonCoordinatesLowRes`. He also indicated that the creation of the new property had to be done by a full time DataCommons person.

Reply:
RE: naming... we already have the "geoJsonCoordinates" property. I was just trying to pick an alternate property name. Any other reasonable name works too.

RE: who can add it... my comment that a DC person should add it was before tjann@ set up the schema directory. you can make a separate change to add it next to https://github.com/datacommonsorg/data/blob/f1a4f35019f87b577049ff39ca9fd9529d2f0bcd/schema/dcschema.mcf#L323-L328

Reply:
@fpernice-google it might be worth coming up with a more specific name than something involving "low res". E.g. today, Guha said that you got 20% (or 20x) reduction, but he might want even more. To leave room for multiple levels, I would propose something more specific. Throwing some ideas out there, but lmk what you come up with: 

douglasPeuckerEpsilon0.01GeoJsonCoordinates
douglasPeucker0.01GeoJsonCoordinates
rdpEpsilon0.01GeoJsonCoordinates

Neither of these seem very idea since RDP is probably not commonly known. geoJsonCoordinatesDouglasPeuckerSimplified0.01?

Reply:
good idea. very slightly prefer geoJsonCoordinates<foo> over <foo>GeoJsonCoordinates.

Reply:
See #284 for final schema.